x-pack/plugins/security_solution/server/endpoint/routes/resolver/utils/fetch.ts

```diff
@@ -89,7 +107,26 @@ export class Fetcher {
    * @param after a cursor to use as the starting point for retrieving alerts
    */
   public async alerts(limit: number, after?: string): Promise<ResolverRelatedAlerts> {
-    return this.doAlerts(limit, after);
+    const query = new AlertsQuery(
```

Comment:
Clean up suggestion by @oatkiller since `alerts` function just called through to `doAlerts` before.